### PR TITLE
Composable finance add dali to rococo

### PR DESCRIPTION
**Pull request by alexkdefi**
Hi!

Please add our Rococo network to apps.

### CODE REVIEW

File: packages/apps-config/src/ui/logos/index.ts [hunk 1]
```diff
@@ -219,6 +221,8 @@ export const chainLogos = Object.entries({
   DataHighway: nodeDataHighway,
   'DataHighway Spreehafen Rococo Parachain Testnet': chainRoccoDataHighway,
   'Dolphin Testnet': nodeDolphin,
+  DaliTestnet: nodeDaliTestnet,
+  'DaliTestnet based on Rococo' : chainRococoDali,
```
Comment (seunlanlege) [inline packages/apps-config/src/ui/logos/index.ts:225]:
```suggestion
  'Dali Testnet (Rococo Relay)' : chainRococoDali,
```

File: packages/apps-config/src/endpoints/testingRelayRococo.ts [hunk 2]
```diff
@@ -531,6 +531,16 @@ export function createRococo (t: TFunction): EndpointOption {
           Zenlink: 'wss://rococo-parachain.zenlink.pro'
         }
       }
+
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRelayRococo.ts:534]:
```suggestion
```

**Review by jacogr — COMMENTED** [review]
One small housekeeping line removal. Apart from that, just ensure that Sean's comment is catered for - the name there needs to match with the chain name (as displayed and as from the RPC)

**Comment by alexkdefi** [comment]
added suggested changes. Thanks!

File: packages/apps-config/src/endpoints/testingRelayRococo.ts [hunk 3]
```diff
@@ -531,6 +531,15 @@ export function createRococo (t: TFunction): EndpointOption {
           Zenlink: 'wss://rococo-parachain.zenlink.pro'
         }
       }
+      {
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRelayRococo.ts:534]: Comma missing on previous line. 

CI will complain about this as well - the entries are expected to be alphabetical.

**Review by jacogr — CHANGES_REQUESTED** [review]
Some CI failures (current and future once that is fixed)

**Review by jacogr — CHANGES_REQUESTED** [review]
See https://github.com/polkadot-js/apps/runs/5771574589?check_suite_focus=true#step:3:697

**Comment by jacogr** [comment]
I'm closing this since the last update has been 21 days ago and it is still failing. Would certainly like seeing this adjusted to pass CI check.

**Pull request closed without merging**